tests/pos/Flags1.scala

```diff
@@ -0,0 +1,376 @@
+import language.implicitConversions
+
+object Flags {
+
+  object opaques {
+
+    /** A FlagSet represents a set of flags. Flags are encoded as follows:
+    *  The first two bits indicate whether a flag set applies to terms,
+    *  to types, or to both.  Bits 2..63 are available for properties
+    *  and can be doubly used for terms and types.
+    */
+    opaque type FlagSet = Long
+    def FlagSet(bits: Long): FlagSet = bits
+    def toBits(fs: FlagSet): Long = fs
+
+    /** A flag set consisting of a single flag */
+    opaque type Flag <: FlagSet = Long
+    private[Flags] def Flag(bits: Long): Flag = bits
+  }
+  export opaques.FlagSet
+
+  type Flag = opaques.Flag
+
+  delegate FlagOps {
+
+    def (x: FlagSet)
```

Comment:
This is the syntax being replaced but it's in the `tests/pos` directory. This should result in a test failure right? Was this committed by accident?

Reply:
> I am thinking about allowing both new and old syntax (the latter fixed wrt newlines) during one release cycle. That gives us time to experiment at large with it before deciding.

Ah I just read this comment - ignore me then.